11ty/definitions/algorithmic-opacity.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Algorithmic Opacity
+slug: algorithmic-opacity
+defined: true
+speech: noun
+---
+
+The effect of an algorithmic system designed in a way that makes difficult to explain why a certain decision was made by it.
```

Comment:
```suggestion
the effect of an algorithmic system designed in a way that makes difficult to explain why a certain decision was made by it.
```

I think here we should avoid defining it in a partially self-referential way? Can we define the "algorithmic" aspect of it to avoid this?

Reply:
Would "when an algorithmic system (a set of instructions, tasks, or processes to accomplish a task) is designed in a way that makes its decision making difficult to explain or understand"

Reply:
the effect of a system (e.g., instructions, tasks, or processes to accomplish a task) that designed in a way that makes its decision making difficult to explain or understand